### DISCUSSION TITLE
Robots in Rhino 7

### DISCUSSION THREAD

**Discussion by Yonahel** [visose/Robots#123] [Q&A] [answered]
Hello Robots Team,

I recently updated my Rhino from Rhino 6 to Rhino 7. On update, Robots had many outdated modules (screenshot attached). I wasn't able to update them through Grasshopper, and was unable to find them on food4rhino. I redid the install process following the Github instructions, but the modules did not appear again. My old scripts are now broken, and I was reaching out to see if I missed a step in the install process that would be causing this issue.
![Robots Grasshopper](https://user-images.githubusercontent.com/4017091/134378378-978a3308-ffbc-49b7-b2ab-ef399875a52a.PNG)

**Comment by visose** [comment]
It looks like it's not installed. You can find the latest version in the Release section here at GitHub.

**Comment by Yonahel** [comment]
Apologies, but as part of my post, I mentioned I redid the install process on Rhino 7 (the same process I followed to install in Rhino 6 and it worked), but it gave the above error when I tried to do so on Rhino 7.

**Comment by visose** [comment]
You probably missed a step installing as you mentioned. The process should be the same for Rhino 7, download the latest release from here: https://github.com/visose/Robots/releases.

Don't use the "Download and Install" button yet, as it will install an old version. As suggested in another issue, I will publish it to the package manager to make the install process more user friendly.

**Comment by Yonahel** [comment]
I did these instructions step by step just now to retry, no success. The only difference from the install process that I did (I also did it for Rhino 6, and it worked) was unzipping the folder instead of "unblocking"  the files through properties - on Windows 10, that's not an option. I also just redid the exact same install on Rhino 6 just now, to test if I'm missing a step, but it worked perfectly on Rhino 6. 

   1.  Download the robots.gha and robots.dll files from the latest release.

    Open the folder that contains the downloaded files, right click on them individually, select properties and click on Unblock.
    Place the files on the Grasshopper components folder. You can find this folder through the Grasshopper menu by clicking on File, Special Folders, Components Folder.
    In your Windows Documents folder, create a subfolder named Robots and place the XML and 3DM files that correspond to one ore more robot libraries.
    Open Rhino, run the Rhino command GrasshopperDeveloperSettings and disable "Memory load *.GHA assemblies using COFF byte arrays".
    Restart Rhino and open Grasshopper. There should be a new tab in Grasshopper named Robots.

**Comment by visose** [comment, marked as answer]
Check the individual extracted files (robots.dll, robots.gha) and see if they have an option to unblock.

**Comment by Yonahel** [comment]
That did work, thank you for your assistance!
